bentoml/_internal/configuration/containers.py

```diff
@@ -174,7 +175,7 @@ def as_dict(self) -> t.Dict[str, t.Any]:
         return self.config
 
 
-@container
+@dataclass
```

Comment:
I thought we stopped using dataclass?

Reply:
@aarnphm it is ok to use dataclass for this use case if it works ok - it does not add extra dependency, right?

Reply:
yep, just for consistency sake